rasa/nlu/training_data/loading.py

```diff
@@ -28,7 +29,6 @@
 WIT = "wit"
 LUIS = "luis"
 RASA = "rasa_nlu"
-UNK = "unk"
```

Comment:
I'd actually rather keep the constant here with the others.

Reply:
It is back